libs/mark-flow-ui/src/components/candidate_contest.tsx

```diff
@@ -392,6 +393,7 @@ export function CandidateContest({
               variant="primary"
               autoFocus
               onPress={closeAttemptedVoteAlert}
+              id={PageNavigationButtonId.NEXT}
             >
               {appStrings.buttonOkay()}
               <AudioOnly>
```

Comment:
Kept these next audio descriptions unchanged as at that point you are focusing on the Okay button